spec/rubocop/cop/capybara/current_path_expectation_spec.rb

```diff
@@ -132,6 +132,29 @@
     RUBY
   end
 
+  it 'registers an offense with `match` with a multi-line string argument' do
+    expect_offense(<<~'RUBY')
+      expect(page.current_path).to match("string/" \
+      ^^^^^^ Do not set an RSpec expectation on `current_path` in Capybara feature specs - instead, use the `have_current_path` matcher on `page`
+                                        "foo/")
+    RUBY
+
+    expect_correction(<<~'RUBY')
+      expect(page).to have_current_path(/string\/foo\//)
+    RUBY
+  end
+
+  it 'registers an offense with `match` with a `command`' do
+    expect_offense(<<~RUBY)
+      expect(page.current_path).to match(`pwd`)
+      ^^^^^^ Do not set an RSpec expectation on `current_path` in Capybara feature specs - instead, use the `have_current_path` matcher on `page`
+    RUBY
+
+    expect_correction(<<~'RUBY')
+      expect(page).to have_current_path("#{`pwd`}")
```

Comment:
Why is the correction like this? Wouldn't it be equivalent to just:
```suggestion
      expect(page).to have_current_path(`pwd`)
```

Reply:
It was an error to pass it as a string. You actually need to pass it as a Regexp to `have_current_path`.
The back-quoted string is then executed as a command and its standard output is given as a string.
So I think it needs to be expanded.